addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -432,7 +432,7 @@ var FormRenderer = BasicRenderer.extend({
     * @returns {integer}
     */
     _renderButtonBoxNbButtons: function () {
-        return [2, 2, 4, 6][config.device.size_class] || 7;
+        return [2, 2, 2, 4, 6][config.device.size_class] || 7;
```

Comment:
I believe this is testable
https://github.com/odoo/odoo/blob/12.0/addons/web/static/tests/views/form_tests.js#L4347

Reply:
Testing was a good idea! I found that the size class between 992px and 1199px can handle 7 buttons